Gemfile.tt

```diff
@@ -59,6 +60,7 @@ group :test do
   gem 'json_matchers' # Validate the JSON returned by your Rails JSON APIs
   gem 'webmock' # Library for stubbing and setting expectations on HTTP requests in Ruby
   gem 'simplecov', require: false # code coverage analysis tool for Ruby
```

Comment:
Should we remove this simplecov @neua-wd 🤔 as we have the `simplecov-json` there? or the simplecov-json is a plugin for the `core simplecov`

Reply:
That's right @andyduong1920, `simplecov-json` is a formatter (the `JSONFormatter` in `simplecov.rb`) :
```ruby
SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter.new([SimpleCov::Formatter::HTMLFormatter, SimpleCov::Formatter::JSONFormatter
```